source/Posets/2ElementPoset.lagda

```diff
@@ -0,0 +1,53 @@
+Ian Ray, 9 August 2023
+
+Constructing the two element poset.
+
+\begin{code}
+
+{-# OPTIONS --safe --without-K --exact-split #-}
+
+open import MLTT.Spartan
+open import MLTT.Two-Properties
+open import UF.FunExt
+open import UF.PropTrunc
+open import UF.Logic
+
+open import UF.Subsingletons
+open import UF.Subsingletons-FunExt
+
+module Posets.2ElementPoset
+  (pt : propositional-truncations-exist)
+  (fe : Fun-Ext)
+  where
+
+open import Locales.Frame pt fe
+
+2-Poset : Poset 𝓤₀ 𝓤₀
+2-Poset = (MLTT.Spartan.𝟚 , (2-ord , 2-ord-is-partial-order))
+  where
+  2-ord : MLTT.Spartan.𝟚 → MLTT.Spartan.𝟚 →  Ω 𝓤₀
+  2-ord ₀ x = (𝟙 , 𝟙-is-prop)
+  2-ord ₁ ₀ = (𝟘 , 𝟘-is-prop)
+  2-ord ₁ ₁ = (𝟙 , 𝟙-is-prop)
```

Comment:
Our convention is to indent inside `where` blocks as well. I have seen some Agda programmers not indenting inside `where` blocks, but we do that.

I'm not sure if this is written down in `CONTRIBUTING.md`.